CHANGES.md

```diff
@@ -43,14 +46,30 @@ To be released.
 
 ### Behavioral changes
 
+ -  Many types became serialized and deserialized better with
+    [`System.Text.Json.JsonSerializer`] as they now have their own
+    [custom converters].  [[#2322]]
+
+     -  An `Address` became represented as a single hexadecimal string in JSON.
+     -  A `BlockHash` became represented as a single hexadecimal string in JSON.
+     -  A `Currency` became represented as an object with values in JSON.
+        Note that it contains its `Hash` and it throws `JsonException`
+        if a JSON object to deserialize has an inconsistent `Hash` with
+        other field values.
+     -  A `FungibleAssetValue` became represented as an object with
+        its `Currency` object and `Quantity` string.
+     -  A `HashDigest` became represented as a single hexadecimal string in
+        JSON.
+     -  A `TxId` became represented as a single hexadecimal string in JSON.
+
 ### Bug fixes
 
 ### Dependencies
 
- -  Upgraded *Bencodex* from [0.4.0][Bencodex 0.4.0] to [0.5.0][Bencodex 0.5.0].
-    [[#2283]]
- -  Upgraded *Bencodex* from [0.5.0][Bencodex 0.5.0] to [0.6.0][Bencodex 0.6.0].
-    [[#2298]]
+ -  Upgraded *Bencodex* from [0.4.0][Bencodex 0.4.0] to [0.6.0][Bencodex 0.6.0].
+    [[#2283]] [[#2298]]
```

Comment:
I'm not sure about the CHANGELOG rule, Shouldn't that be like the below code?

```suggestion
    [[#2283], [#2298]]
```

Reply:
As 0.42.0 was already released, I gave up this adjustment at all…